examples/gallery/src/mapbox-layer.html

```diff
@@ -11,11 +11,9 @@
 
     const {MapboxLayer, ScatterplotLayer} = deck;
 
-    mapboxgl.accessToken = '<mapbox-access-token>';
-
     const map = new mapboxgl.Map({
       container: document.body,
-      style: 'mapbox://styles/mapbox/light-v9',
```

Comment:
Since it already requires the token, let's keep the basemap from mapbox

Reply:
Fixed: e172c00